pkg/operator/ceph/pool/config.go

```diff
@@ -60,11 +58,12 @@ func (r *ReconcileCephBlockPool) createBootstrapPeerSecret(cephBlockPool *cephv1
 
 	// Create Secret
 	logger.Debugf("store rbd-mirror bootstrap token in a Kubernetes Secret %q", s.Name)
-	_, err = r.context.Clientset.CoreV1().Secrets(cephBlockPool.Namespace).Create(ctx, s, metav1.CreateOptions{})
+	_, err = k8sutil.CreateOrUpdateSecret(r.context.Clientset, s)
 	if err != nil && !kerrors.IsAlreadyExists(err) {
-		return opcontroller.ImmediateRetryResult, errors.Wrapf(err, "failed to create rbd-mirror bootstrap peer %q secret", s.Name)
+		return opcontroller.ImmediateRetryResult, errors.Wrapf(err, "failed to create or update rbd-mirror bootstrap peer %q secret", s.Name)
 	}
 
+	logger.Infof("successfully created bootstrap peer token secret for pool %q", cephBlockPool.Name)
```

Comment:
Should it be debug?
```suggestion
	logger.Debugf("successfully created bootstrap peer token secret for pool %q", cephBlockPool.Name)
```

Reply:
I believe it's useful to print success after the command. Also, we recently discussed/agreed we should do this more in the client package.